tests/test_outliers/test_check_estimator_outliers.py

```diff
@@ -21,3 +23,19 @@ def test_check_estimator_from_feature_engine(estimator):
     if estimator.__class__.__name__ == "ArbitraryOutlierCapper":
         estimator.set_params(max_capping_dict={"var_1": 10})
     return check_feature_engine_estimator(estimator)
+
+
+@pytest.mark.parametrize("transformer", _estimators)
+def test_transformers_in_pipeline_with_set_output_pandas(transformer):
```

Comment:
This test will fail when

```python
transformer = Winsorizer(add_indicators=True)
```

Reply:
Thank you! I missed it.

I had to make a workaround that doesn't make me happy to modify the BaseOutlier.

I need to look at class inheritance at some point. It's getting out of hand.

Thanks a lot for flagging this one!